tests/csm/rest/test_csm_users.py

```diff
@@ -4655,6 +4656,95 @@ def test_28516(self):
         self.csm_obj.check_expected_response(response, HTTPStatus.OK)
 
 
+    @pytest.mark.skip(reason="not_in_main_build_yet")
+    @pytest.mark.sanity
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.tags('TEST-44783')
+    def test_44783(self):
+        """
+        Verify same token is reused in case of existing active session
+        for multiple user login from same client
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+
+        for username, password in self.users_dict.items():
+            self.log.info("Step 1: Login wtih %s %s and Get Header", username, password)
+            header = self.csm_obj.get_headers(username, password)
+            self.log.info("Step 2: Store Authorization Token")
+            init_token = header['Authorization']
+            self.log.info("Step 3: Login Again and Get Header")
+            header = self.csm_obj.get_headers(username, password)
+            self.log.info("Step 4: Store Authorization Token")
+            new_token = header['Authorization']
+            self.log.info("Step 5: Check Bearer token should be same")
+            self.log.info("init_token = %s new_token = %s ", init_token, new_token)
+            assert (init_token == new_token),"unexpected token mismatch"
+
+
+    @pytest.mark.skip(reason="not_in_main_build_yet")
+    @pytest.mark.sanity
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.tags('TEST-44785')
+    def test_44785(self):
+        """
+        Verify different token is generated when user logout and login again
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+
+        for username, password in self.users_dict.items():
+            self.log.info("Step 1: Login wtih %s %s and Get Header", username, password)
+            header = self.csm_obj.get_headers(username, password)
+            self.log.info("Step 2: Store Authorization Token")
+            init_token = header['Authorization']
+            self.log.info("Step 3: Logout user session")
+            response = self.csm_obj.csm_user_logout(header)
+            self.csm_obj.check_expected_response(response, HTTPStatus.OK)
```

Comment:
Can use simple assert here? or maybe we can reason if this function is adding any value.
My personal preference is we should avoid asserts in libs as it will fail in the lib and we will have to check complete log to know which step actually failed.

Reply:
this was used in the same file in all similar tests so